### ISSUE TITLE
RSS feed of (all) feeds is required

### ISSUE THREAD

**Issue by noamoss** [niryariv/opentaba-client#116]
for many usages, i.e. a combined Telegram bot.

(see the #opentaba_bot for example)

**Comment by niryariv** [comment]
@noamoss by feed of all feeds do you mean a single feed combining all municipalities?

**Comment by noamoss** [comment]
@niryariv yep. which will extend automatically when we add new municaplities.

**Comment by niryariv** [comment]
The "automatically" part is a bit tricky. I'm still not clear on the potential audience. Who would be interested in a feed of all munis?

**Comment by noamoss** [comment]
@niryariv developers (like me), analysers, social organizations and reporters

**Comment by niryariv** [comment]
Can you provide a usage scenario? This is non-trivial to implement automatically (but trivial to implement manually, with some RSS aggregator if these still exist)

**Comment by noamoss** [comment]
1) a general feed which enables "heavy users" to follow after updates at the national with one simple action.

2) a developer who wants to integrate future municipalities in his/her service.
